.travis.yml

```diff
@@ -12,7 +12,7 @@ matrix:
     - php: 5.5
       env: COMPOSER_FLAGS="--prefer-lowest"
     - php: 7.0
-      env: COMPOSER_FLAGS="--prefer-stable" SYMFONY_VERSION=2.7.*
+      env: COMPOSER_FLAGS="--prefer-stable" SYMFONY_VERSION=2.8.*
   fast_finish: true
```

Comment:
We're currently triggering 5 builds Travis... but no one is built using SF3.

I propose to replace the whole travis.yml file with the following (which will trigger 6 builds and covers all our requirements IMHO):

```
language: php

php:
  - 5.5
  - 7.0

env:
  - SYMFONY_VERSION="2.8.*"
  - SYMFONY_VERSION="3.1.*"
  - COMPOSER_FLAGS="--prefer-lowest"

matrix:
  fast_finish: true

before_install:
  - phpenv config-rm xdebug.ini
  - bash travis/install_amqp.sh
  - composer self-update
  - if [ "$SYMFONY_VERSION" != "" ]; then composer require symfony/symfony:${SYMFONY_VERSION} --no-update; fi

install: composer update --prefer-dist $COMPOSER_FLAGS

script: bin/atoum
```